sdks/java/container/boot.go

```diff
@@ -148,9 +149,20 @@ func main() {
 		"-Xmx" + strconv.FormatUint(heapSizeLimit(info), 10),
 		"-XX:-OmitStackTraceInFastThrow",
 		"-cp", strings.Join(cp, ":"),
-		"org.apache.beam.fn.harness.FnHarness",
 	}
 
+	enable_google_cloud_profiler := strings.Contains(options, "enable_google_cloud_profiler")
+	if enable_google_cloud_profiler {
+		if metadata := info.GetMetadata(); metadata != nil {
+			if jobName, nameExists := metadata["job_name"]; nameExists {
+				if jobId, idExists := metadata["job_id"]; idExists {
+					args = append(args, fmt.Sprintf("-agentpath:/opt/google_cloud_profiler/profiler_java_agent.so=-cprof_service=%v,-cprof_service_version=%v", jobName, jobId))
```

Comment:
If we expect that job_name/id are set but they aren't actually in metadata, should we add a log line for why we don't actually enable cloud profiling? e.g. "given option enable_google_cloud_profiler but not enabling because job metadata not found"

Reply:
done.